blended-measurement/blended-measurement.tex

```diff
@@ -0,0 +1,685 @@
+\documentclass[10pt]{article}
+
+% \setcitestyle{numbers}
+\usepackage[numbers]{natbib}
+\usepackage[english]{babel}
+\usepackage[utf8x]{inputenc}
+\usepackage{amsmath}
+\usepackage{graphicx}
+\usepackage{longtable}
+\usepackage[]{hyperref}
+\usepackage{comment}
+\usepackage{xspace}
+\usepackage[usenames]{color}
+\usepackage{datetime}
+\usepackage{bm}
+
+\title{Measurement of Blended Objects in LSST\\
+{\author{
+    Jim Bosch\\
+    {\em for LSST Data Management}
+}}}
+
+\begin{document}
+
+\maketitle
+
+\section{Introduction}
+
+Most LSST objects will overlap one or more of its neighbors enough to affect
```

Comment:
Would be good to cite the requirements (i.e. DPDD) for what this measurement is intending to produce.